Wiki/BlankConfig/index.md

```diff
@@ -0,0 +1,44 @@
+---
+layout : wiki
+title : BlankConfig
+type : config
+weight : 500
+description : |
+  This is a description of how to configure Fvwm3 from scratch with a blank config file
+---
+
+* TOC
+{:toc}
+
+# Starting Fvwm3 with a Blank Config File
+
+Fvwm has been written with literally hundreds of options for most (if not all) aspects of window management.
+
+In order for windows to appear in the first place, there needs to be a root area (which, in turn, needs to "look like something"); a window needs to either have decorations or no decorations; if it has decorations, they need to be defined and styled; once a window is opened, it needs to open with some kind of geometry somewhere over the root area, etc...
+
+Imagine you need to define all these options _before_ you can make use of your window manager -- you'd be configuring for quite some time. To provide a working Fvwm3, all the necessary options have a default setting so that you can fire up Fvwm3 from the console and begin using it as your window management tool.
```

Comment:
```suggestion
Imagine you need to define all these options _before_ you can make use of your window manager -- you'd be configuring it for quite some time. To provide a working Fvwm3, all the necessary options have a default setting so that you can fire up Fvwm3 from the console and begin using it as your window management tool.
```